src/amo/components/SectionLinks/index.js

```diff
@@ -97,6 +98,22 @@ export class SectionLinksBase extends React.Component<InternalProps> {
 
     return (
       <ul className={makeClassName('SectionLinks', className)}>
+        {withBlogUI && (
+          <li>
+            <Link
+              className={makeClassName(
+                'SectionLinks-link',
+                'SectionLinks-blog',
+                'SectionLinks-link--active',
+              )}
+              href="/blog/"
+              prependClientApp={false}
+              prependLang={false}
```

Comment:
I read this as the link will be rendered as `https://addons.mozilla.org/blog/` (on prod), without the lang and app (e.g. without `/en-US/firefox/`), but blog isn't defined in `validClientAppUrlExceptions` and `validLocaleUrlExceptions` - should it be?

Reply:
ah, it will probably be needed in the near future indeed. I filed https://github.com/mozilla/addons-frontend/issues/10388

Reply:
mmm, this won't be needed right now actually. This link is only rendered when the `Header` is rendered for the blog, so not for the frontend pages. That being said, the issue is still valid because we'll want to have a link to this new blog (which is indeed served at `https://addons.mozilla.org/blog/` on prod).